### PR TITLE
dashboard: add integration test for authority

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve?

1. minor update on archive API, only manager role can delete archive experiment
2. add integration test

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/script_test.yml [hunk 1]
```diff
@@ -28,4 +28,4 @@ jobs:
 
       - name: Run integration test
         run: |
-          bash test/integration_test/run.sh
\ No newline at end of file
+          bash test/integration_test/run.sh network
```
Comment (WangXiangUSTC) [inline .github/workflows/script_test.yml:31]: this test just used to test the script, so just run one case is ok

File: .github/workflows/integration_test.yml [hunk 2]
```diff
@@ -26,7 +26,10 @@ jobs:
         run: |
           export CLUSTER="chart-testing"
           hack/local-up-chaos-mesh.sh
-
+          kubectl set env deployment/chaos-dashboard SECURITY_MODE=true -n chaos-testing
+          sleep 5
+          kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333 &
+
```
Comment (g1eny0ung) [inline .github/workflows/integration_test.yml:32]: Should the integration test influence the `release-1.0` branch? This workflow will run when the `v1` branch changed (v1 has no support of auth).

File: .github/workflows/integration_test.yml [hunk 3]
```diff
@@ -26,7 +26,10 @@ jobs:
         run: |
           export CLUSTER="chart-testing"
           hack/local-up-chaos-mesh.sh
-
+          kubectl set env deployment/chaos-dashboard SECURITY_MODE=true -n chaos-testing
+          sleep 5
```
Comment (g1eny0ung) [inline .github/workflows/integration_test.yml:30]: Can u explain why it needs to sleep for 5s? I am a little confused about this.

Comment (WangXiangUSTC) [reply .github/workflows/integration_test.yml:30]: before add `sleep 5` will get error:

```
an error occurred forwarding 2333 -> 2333: error forwarding port 2333 to pod a98ee83ece52039639db7b1ee0d1811b1380ad10e5bc97b3dce3cdfb5de80e2b, uid : failed to find sandbox "a98ee83ece52039639db7b1ee0d1811b1380ad10e5bc97b3dce3cdfb5de80e2b" in store: does not exist
```

maybe the pod is not finished the upgrade, so just sleep some seconds

File: .github/workflows/integration_test.yml (hunk 2 above)
Comment (WangXiangUSTC) [reply .github/workflows/integration_test.yml:30]: you are right, I removed the `release-1.0`

File: .github/workflows/integration_test.yml (hunk 3 above)
Comment (g1eny0ung) [reply .github/workflows/integration_test.yml:28]: Got it!

**Review by g1eny0ung — APPROVED** [review]
LGTM

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**